src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -127,23 +131,36 @@ public function getStyle($name)
     }
 
     /**
-     * {@inheritdoc}
+     * @return int
      */
-    public function format($message)
+    public function getDefaultWrapCutOption(): int
```

Comment:
not really needed is it

Reply:
A protected property that you can only write? It looks a little bit bad concept. A property what you can only read, it could be justified, but a property without reading option, not really.

Reply:
im skeptical about the setter too :)

setters/getters can live independently IMHO, it's not needed for every property by definition or so. If we dont have reason to add  a getter/settter i would avoid it (it introduces extra state).

In this case i'd prefer passing the cut options to formatAndWrap(), i dont see a real reason to control this default behavior.